server/db/models/products-model.js

```diff
@@ -0,0 +1,22 @@
+const connection = require('../database')
+const { Sequelize: {STRING, DOUBLE} } = connection
+
+const Product = connection.define('product', {
+    name: {
+        type: STRING,
+        allowNull: false,
+        notEmpty: true,
+    },
+    color: {
```

Comment:
Only iffy about the color part for now but I think it should be ok since we don't know what we are selling @ydhan18

Reply:
It didn't know what product we were selling so I thought I'd have at least one attribute. We can change it later on @seloboa